internal/pkg/template/templates/workloads/services/rd-web/manifest.yml

```diff
@@ -33,6 +33,11 @@ cpu: {{.InstanceConfig.CPU}}
 # Amount of memory in MiB used by the task.
 memory: {{.InstanceConfig.Memory}}
 
+# # Connect your App Runner service to your environment's VPC.
+# network:
+#   vpc:
+#     placement: 'private'
```

Comment:
We don't have single quotes in other commented out strings. Should we be consistent?

Reply:
Good catch! updated